velox/functions/lib/Re2Functions.cpp

```diff
@@ -400,6 +443,17 @@ bool matchPrefixPattern(
       std::memcmp(input.data(), pattern.data(), length) == 0;
 }
 
+bool matchRelaxedPrefixPattern(
+    StringView input,
+    const std::vector<SubPatternMetadata>& subPatterns,
+    size_t length) {
+  if (input.size() < length) {
```

Comment:
we can remove this check since it is already included in matchRelaxedFixedPattern 

we can remove this function altogether I think.